docs/CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- New config variable, `JobPipeline.MaxSuccessfulRuns` caps the total number of
+  saved completed runs per job. This is done in response to the `pipeline_runs`
+  table potentially becoming large, which can cause performance degradation.
+  The default is set to 10,000. You can set it to 0 to disable run saving
+  entirely.
```

Comment:
Let's start a new section up top so that we don't have to remember to pick this back out when merging 1.11.0 back in.

Reply:
Easiest way to do this is just merge 1.11.0-rc2 back into develop

Reply:
We necessarily have a limbo period before releases are merged back in that has to be accounted for. We have a new release process to do so, by pre-arranging this very separation, and it will start next release.

Reply:
It's fixed.

Reply:
This is still listed under 1.11.0

Reply:
Fixed